aten/src/ATen/native/native_functions.yaml

```diff
@@ -1919,3 +1919,11 @@
 
 - func: meshgrid(TensorList tensors) -> TensorList
   variants: function
+
+- func: gather(Tensor self, int64_t dim, IndexTensor index) -> Tensor
+  variants:
```

Comment:
`variants` are `function` and `method` by default.

Reply:
Just fixed, thanks!